### DISCUSSION TITLE
Issue with Nginx Proxy Manager

### DISCUSSION THREAD

**Discussion by Gangstrocity** [Athou/commafeed#1186] [Q&A]
I've got CommaFeed set up and it works great on my home network. I'm trying to host it so I can access it when I'm away from my network and I use Nginx Proxy Manager for this. I am successfully using NPM to access 8 other services on my network, but I can't get commaFeed to work.

It actually does work occasionally for about 2 minutes, and then if I try to refresh or click on a different feed it will error out and give me a 404. Sometimes it will sit on a loading screen and then give me a 404. Are there any particular settings I need to use to make CommaFeed work with NPM?

**Comment by Athou** [comment]
> Are there any particular settings I need to use to make CommaFeed work with NPM?

I don't know Nginx Proxy Manager so I can't help you :( I don't think you need to do anything special except proxy pass the port 8082.

**Reply by Gangstrocity** [reply]
Yeah NPM is pretty simple. That's exactly what I'm doing.The link works inside my network, but from outside I get stuck at loading or I get 404.
